modules/apps/commerce/commerce-product-definitions-web/src/main/resources/META-INF/resources/edit_cp_definition_visibility.jsp

```diff
@@ -76,6 +76,10 @@ Map<String, String> contextParams = HashMapBuilder.<String, String>put(
 			"channelDataSetId", CommerceProductFDSNames.PRODUCT_CHANNELS
 		).put(
 			"channelItemSelectorURL", cpDefinitionsDisplayContext.getChannelItemSelectorUrl()
+		).put(
+			"checkedAccountGroupIds", cpDefinitionsDisplayContext.getCheckedCommerceAccountGroupIds()
+		).put(
+			"checkedChannelIds", cpDefinitionsDisplayContext.getCheckedCommerceChannelIds()
```

Comment:
is this "checkedChannelIds" correct? on other points is called checkedCommerceChannelIds